libgalois/include/katana/Properties.h

```diff
@@ -507,17 +507,17 @@ struct StructProperty
 
 
 template <typename T>
-class VectorRef {
+class ArrayRef {
   public:
     using value_type = T;
     using iterator = value_type*;
     using const_iterator = const value_type*;
 
-    VectorRef(T* ptr=NULL, size_t size=1): data_(ptr), size_(size){}
+    ArrayRef(T* ptr=NULL, size_t size=1): data_(ptr), size_(size){}
   
-    void setPtr(T* ptr) {data_ = ptr;}
-    void setSize(size_t size) {size_ = size;}
-    bool isDataNull() {return data_ == NULL;}
+    void SetPtr(T* ptr) {data_ = ptr;}
+    void SetSize(size_t size) {size_ = size;}
+    bool IsDataNull() {return data_ == NULL;}
```

Comment:
These setters shouldn't be needed unless I'm missing something

Reply:
They are not needed anymore. I removed them.